docs/src/main/asciidoc/gradle-config.adoc

```diff
@@ -8,43 +8,42 @@ https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 include::./attributes.adoc[]
 
 // tag::repositories[]
-Quarkus Gradle plugin is not yet published to the https://plugins.gradle.org/[Gradle Plugin Portal],
-so you need to add the following at the top of your './settings.gradle' file:
+The Quarkus Gradle plugin is published to the https://plugins.gradle.org/plugin/io.quarkus[Gradle Plugin Portal].
+
+To use it, add the following to your `build.gradle` file:
+
 [source, groovy, subs=attributes+]
 ----
-pluginManagement {
-    repositories {
-        mavenCentral()
-        gradlePluginPortal()
-    }
-    resolutionStrategy {
-        eachPlugin {
-            if (requested.id.id == 'io.quarkus') {
-                useModule("io.quarkus:quarkus-gradle-plugin:${requested.version}")
-            }
-        }
-    }
+plugins {
+    id 'java'
+    id 'io.quarkus'
 }
 ----
 
-Or, if you use the Gradle Kotlin DSL, you need to add the following at the top of your './settings.gradle.kts' file:
-[source, kotlin, subs=attributes+]
+You also need to add the following at the top of your `settings.gradle` file:
+[source, groovy, subs=attributes+]
 ----
 pluginManagement {
     repositories {
         mavenCentral()
         gradlePluginPortal()
     }
-    resolutionStrategy {
-        eachPlugin {
-            if (requested.id.id == "io.quarkus") {
-                useModule("io.quarkus:quarkus-gradle-plugin:${requested.version}")
-            }
-        }
+    plugins {
+      id 'io.quarkus' version "${quarkusPluginVersion}"
     }
 }
 ----
 
-This won't be necessary anymore once the Quarkus Gradle plugin is published in the Gradle plugin portal.
+NOTE:: the `plugins{}` method is not supported in Gradle 5.x. In this case make sure to explicitly declare the plugin version in the `build.gradle` file like the example below:
```

Comment:
Can we perhaps add in `settings.gradle` after `plugins {}`? I know it should be obvious, but it through me off at first (probably because I am so dense when it comes to Gradle :P)

Reply:
Fixed as requested :)

![image](https://user-images.githubusercontent.com/54133/70326513-7aafac80-1813-11ea-973e-988b6873c77b.png)

Reply:
LOLOLOLOL